scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1461,109 @@ def __getitem__(self, indexer):
+            Covariance matrix of the three component error vector of the Euler
+            angles describing the `estimated_rotation`.
+
+        References
+        ----------
+        .. [1] F. Landis Markley,
+                “Attitude determination using vector observations: a fast
+                optimal matrix algorithm,” Journal of Astronautical Sciences,
+                Vol. 41, No.2, 1993, pp. 261-280.
+        """
+        outvecs = np.asarray(a)
+        if outvecs.ndim != 2 or outvecs.shape[-1] != 3:
+            raise ValueError("Expected input `a` to have shape (N, 3), "
+                             "got {}".format(outvecs.shape))
+        vectors = np.asarray(b)
+        if vectors.ndim != 2 or vectors.shape[-1] != 3:
+            raise ValueError("Expected input `b` to have shape (N, 3), "
+                             "got {}.".format(vectors.shape))
+
+        if outvecs.shape != vectors.shape:
+            raise ValueError("Expected inputs `a` and `b` to have same shapes"
+                             ", got {} and {} respectively.".format(
+                                outvecs.shape, vectors.shape))
+
+        if vectors.shape[0] == 1:
+            raise ValueError("Rotation cannot be estimated using a single "
+                             "vector.")
+
+        if weights is None:
+            weights = np.ones(vectors.shape[0])
+        else:
+            weights = np.asarray(weights)
+            if weights.ndim != 1:
+                raise ValueError("Expected `weights` to be 1 dimensional, got "
+                                 "shape {}.".format(weights.shape))
+            if weights.shape[0] != vectors.shape[0]:
+                raise ValueError("Expected `weights` to have number of values "
+                                 "equal to number of input vectors, got "
+                                 "{} values and {} vectors.".format(
+                                    weights.shape[0], vectors.shape[0]))
+        weights = weights / scipy.linalg.norm(weights)
+
+        if not normalized:
+            outvecs = outvecs / scipy.linalg.norm(outvecs, axis=1)[:, None]
+            vectors = vectors / scipy.linalg.norm(vectors, axis=1)[:, None]
+
+        B = np.einsum('ji,jk->ik', weights[:, None] * outvecs, vectors)
+        u, s, vh = np.linalg.svd(B)
+
+        zeta = (s[0]+s[1]) * (s[1]+s[2]) * (s[2]+s[0])
+        # TODO: Maybe change to close to zero?
+        if zeta == 0:
+            raise ValueError("Three component error vector has infinite "
+                             "covariance. It is impossible to determine the "
+                             "rotation uniquely.")
+
+        C = np.dot(u, vh)
+        kappa = s[0]*s[1] + s[1]*s[2] + s[2]*s[0]
+        # For normalized weights with sum(w_i) = 1, the constant factor
+        # lambda_0 * sigma_{tot}^2 equals unity. So we normalize the weights
+        # and return the matrix as is.
+        cov = (kappa * np.eye(3) + np.dot(B, B.T)) / zeta
```

Comment:
I think I'm almost figured 08t what we should do and how to explain what is covariance or "sensitivity matrix". Will write tomorrow about it.